packaging/tools/makepkg.sh

```diff
@@ -69,25 +69,30 @@ if [ "$pagMode" == "lite" ]; then
   bin_files="${build_dir}/bin/${serverName} ${build_dir}/bin/${clientName} ${script_dir}/remove.sh ${script_dir}/startPre.sh ${build_dir}/bin/taosBenchmark "
   taostools_bin_files=""
 else
-
-  wget https://github.com/taosdata/grafanaplugin/releases/latest/download/TDinsight.sh -O ${build_dir}/bin/TDinsight.sh \
+  if [ "$verMode" == "cloud" ]; then
+    taostools_bin_files=" ${build_dir}/bin/taosdump \
```

Comment:
I guess no need to include taosdump in the cloud instances if we have no interface to download files.

Reply:
OK, I will correct it.

Reply:
@sangshuduo have updated the code, please review again.